gapis/memory/pool.go

```diff
@@ -145,6 +150,14 @@ func (m *Pools) Count() int {
 	return len(m.pools)
 }
 
+// SetOnCreate sets the OnCreate callback and invokes it for every pool already created.
+func (m *Pools) SetOnCreate(onCreate func(PoolID, *Pool)) {
```

Comment:
The `invokes it for every pool already created.` bit doesn't look used, so I'd recommend killing this function and just switch over to using the `OnCreate` field directly.